src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/global.yml

```diff
@@ -201,21 +204,27 @@ Resources:
               - sts:AssumeRole
           - Effect: Allow
             Sid: "AssumeRoleLambda"
+            Condition:
+              StringEquals:
+                'aws:PrincipalArn': !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:role/adf-lambda-role
             Principal:
               AWS:
-                - !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:role/adf-lambda-role
+                - !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:root
             Action:
               - sts:AssumeRole
           - Effect: Allow
             Sid: "AssumeRole"
             Principal:
               AWS:
-                - !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:role/adf-codepipeline-role
+                - !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:root
             Action:
               - sts:AssumeRole
             Condition:
-              ArnEquals:
-                "aws:SourceArn": !Sub "arn:${AWS::Partition}:codepipeline:${AWS::Region}:${DeploymentAccountId}:*"
+              Fn::And:
```

Comment:
This doesn't look right here. 
Are you trying to add both conditions in an AND fashion?